StreamingAnalytics/LowLatencySample/Temperature-Epl/monitors/TemperatureEventMonitor.mon

```diff
@@ -0,0 +1,33 @@
+package lowlatency;
+using com.apama.aggregates.mean;
+/** If average temperature in 5 minutes is greater than 80 C, send a shutdwon alert  */
+
+event Temperature {
+	float temperature;
+}
+
+event ShutDown {
+	string shutdownMessage;
+}
+monitor TempEventMonitor {
+	
+	action onload() {
+		monitor.subscribe("in");
+		monitor.subscribe("mqtt:sensors/temperature");
+		monitor.subscribe("mqtt:alerts/temperature");
+		stream<Temperature> temperatures := all Temperature();
+		from temp in temperatures within 300.0
```

Comment:
300.0 should be put into a monitor-level constant with a meaningful name. (see Ashish's PR for an example)

Reply:
Yes, made a monitor level constant for 300.0 .